doc/classes/Viewport.xml

```diff
@@ -144,7 +144,7 @@
 		<method name="gui_is_dragging" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the viewport is currently performing a drag operation.
+				Returns [code]true[/code] if a drag operation is currently performing, that can drop in this viewport.
```

Comment:
```suggestion
				Returns [code]true[/code] if a drag operation is currently ongoing and where the drop action could happen in this viewport.
```

Reply:
Thanks for the code-style and the comment corrections. I have included them.